sdk/src/Services/SQS/Custom/Model/GetQueueAttributesResponse.extensions.cs

```diff
@@ -178,6 +172,28 @@ public string Policy
             }
         }
 
+        /// <summary>
+        /// Whether or not the queue is a First-In-First-Out (FIFO) queue
+        /// </summary>
+        public bool? FifoQueue
+        {
+            get
+            {
+                return getAttributeValue(SQSConstants.ATTRIBUTE_FIFO_QUEUE)?.ToUpper().Equals("TRUE");
+            }
+        }
+
+        /// <summary>
+        /// Whether or not the First-In-First-Out (FIFO) queue has content based deduplication enabled
```

Comment:
While working with `bool?` is more complicated for the end-user, it is the more correct type to use here, since this attribute has three states: ContentBasedDeduplication can be on or off for a FIFO queue, or undefined for a non-FIFO queue. So I still think `bool?` is appropriate here. (Note that FifoQueue does not have this triple-state issue, hence why `bool` works there.)